### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by nebolax**

### CODE REVIEW

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -546,6 +546,7 @@ In the advanced section of the backend settings you can also modify the followin
 - **Main Loop sleep**: This is the amount of seconds that the main loop of rotki sleeps for. It is set to 20 seconds by default.
 - **Max log size**: This is the maximum size in megabytes all logs of a single run can have.
 - **Max num of log files**: This is the maximum number of backup (rotated) logs a single run can have.
+- **Instructions per SQLite context switch**: This is the setting to specify how often database context should be switched. Setting it to 0 means making the database synchronous.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:549]: Use the same text I have in the UI

```suggestion
- **Instructions per SQLite context switch**: This is the setting to specify Specify after how many sqlite VM instructions to context switch between cooperative threads. Set to 0 to disable async DB access.
```

**Review by LefterisJP — COMMENTED** [review]
Also please update the image  `images/rotki_backend_settings.png` and make the PR on bugfixes

File: docs/usage_guide.rst (hunk 1 above)
Comment (nebolax) [reply docs/usage_guide.rst]: Done

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -546,6 +546,7 @@ In the advanced section of the backend settings you can also modify the followin
 - **Main Loop sleep**: This is the amount of seconds that the main loop of rotki sleeps for. It is set to 20 seconds by default.
 - **Max log size**: This is the maximum size in megabytes all logs of a single run can have.
 - **Max num of log files**: This is the maximum number of backup (rotated) logs a single run can have.
+- **Instructions per SQLite context switch**: This is the setting to specify Specify after how many sqlite VM instructions to context switch between cooperative threads. Set to 0 to disable async DB access.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:549]:
```suggestion
- **Instructions per SQLite context switch**: This is the setting to specify after how many sqlite VM instructions to context switch between cooperative threads. Set to 0 to disable async DB access.
```

**Review by LefterisJP — APPROVED** [review]
lgtm

**Pull request merged**